Tests/Unit/Mail/MailerHelperTest.php

```diff
@@ -0,0 +1,210 @@
+<?php
+
+
+namespace Sulu\Bundle\FormBundle\Tests\Unit\Mail;
+
+
+use PHPUnit\Framework\MockObject\MockObject;
+use PHPUnit\Framework\TestCase;
+use Prophecy\Prophecy\ObjectProphecy;
+use Psr\Log\LoggerInterface;
+use Sulu\Bundle\FormBundle\Mail\MailerHelper;
+use Symfony\Component\HttpFoundation\File\File;
+use Symfony\Component\HttpFoundation\File\UploadedFile;
+use Symfony\Component\Mailer\MailerInterface;
+use Symfony\Component\Mime\Email;
+use Symfony\Component\Mime\Address;
+
+/**
+ * Class MailerHelperTest
+ * This tests the MailerHelper class that sends an email using the symfony mailer component.
+ *
+ * The MailerHelper class has 3 protected methods that could be overwritten to change the mail message.
```

Comment:
I will currently not support this. If somebody want to hook in he should use the official symfony way, listening on events of symfony mailer or create a swift mailer plugin.